.circleci/images/integration/config_linux4.17_amd64.txt

```diff
@@ -67,8 +67,10 @@ CONFIG_KERNEL_XZ=y
 CONFIG_DEFAULT_HOSTNAME="(none)"
 CONFIG_SWAP=y
 # CONFIG_SYSVIPC is not set
+# CONFIG_POSIX_MQUEUE is not set
```

Comment:
shouldn't that remain the minimum kernel?

Reply:
That file's comment is misleading and should be updated because it contains non-minimal things such as vfat and /dev/io. Now it's more of the minimal config for running the integration tests.